fe/src/app/service/verify/page.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import PdfDropzone from "@/components/PdfDropzone/PdfDropzone";
```

Comment:
This page could be made as a default route after signing if the role is `service` in this PR or maybe with a separate task to handle service's flows.